js/coinex.js

```diff
@@ -36,9 +36,9 @@ module.exports = class coinex extends Exchange {
                 'fetchFundingRateHistory': true,
                 'fetchFundingRates': false,
                 'fetchIndexOHLCV': false,
-                'fetchLeverage': undefined,
-                'fetchLeverageTiers': undefined,
-                'fetchMarketLeverageTiers': undefined,
+                'fetchLeverage': false,
+                'fetchLeverageTiers': true,
+                'fetchMarketLeverageTiers': false,
```

Comment:
set this to emulated, it's inherited from base.exchange

Reply:
fetchMarketLeverageTiers is currently returning undefined for Coinex, I think we should set this to emulated once we resolve the issue and this PR is ready to merge